### PR TITLE
Automationhub fixes

### PR DESCRIPTION
**Pull request by t-woerner**
- yamllint: Fix EXAMPLE issues
- shellcheck: Double quote to prevent globbing and word splitting
- Use `ansible.module_utils.six` instead of `six`
- Add missing whitespace around arithmetic operator
- ipaclient_get_facts: Fix closing bracket does not match indentation
- Remove "’" from yaml files
- Replace asserts with raise AssertionError
- Remove non-module shebang
- Add __future__ imports and __metaclass__ for automationhub

### CODE REVIEW

File: plugins/doc_fragments/ipamodule_base_docs.py [hunk 1]
```diff
@@ -20,6 +20,10 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
```
Comment (rjeffman) [inline plugins/doc_fragments/ipamodule_base_docs.py:25]: This is a note for another PR: it seems this should be included in the `nem_module` templates, shouldn't it?

File: plugins/module_utils/ansible_freeipa_module.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def parse(version_str):
 
     import socket
     import base64
-    import six
+    from ansible.module_utils import six
```
Comment (rjeffman) [inline plugins/module_utils/ansible_freeipa_module.py:106]: This should also be added to the `new_module` templates (in another PR).

**Comment by rjeffman** [comment]
Some linter issues should have been caught by our CI. I'll look at them so these problems don't show again in the future.

File: plugins/doc_fragments/ipamodule_base_docs.py (hunk 1 above)
Comment (t-woerner) [reply plugins/doc_fragments/ipamodule_base_docs.py:25]: Done

File: plugins/module_utils/ansible_freeipa_module.py (hunk 2 above)
Comment (t-woerner) [reply plugins/module_utils/ansible_freeipa_module.py:106]: Done

File: plugins/doc_fragments/ipamodule_base_docs.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
 # -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (t-woerner) [reply plugins/doc_fragments/ipamodule_base_docs.py:1]: Done

**Comment by varunmylaraiah** [comment]
Executed RHEL-8.6 pipeline with Automationhub fixes. https://gitlab.cee.redhat.com/mvarun/trigger-tool/-/pipelines/1495039

**Pull request closed without merging**